content/en/blog/news/roadmap-july-2019.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "OpenCue roadmap"
+linkTitle: "OpenCue roadmap"
+date: 2019-07-08
+description: "Learn more about the OpenCue roadmap"
+---
+
+If you're attending SIGGRAPH 2019 in Los Angeles, you can learn more about
+the roadmap plans for OpenCue at an hour long Birds of a Feather session on
+July 30.
+
+The Academy Software Foundation (ASWF) will provide boxed lunches for this
+session.
```

Comment:
This text about lunch isn't relevant anymore now that the event is at 10AM.

(It still appears on the ASWF page but I've asked them to fix this.)

Reply:
Thanks, removed from the post.